tests/MongoDB.Driver.Tests/Specifications/change-streams/ChangeStreamTestRunner.cs

```diff
@@ -402,12 +417,28 @@ private void AssertChangeStreamDocumentPropertyValuesAgainstBackingDocument(Chan
                 clusterTime.Should().BeNull();
             }
 
+            var operationType = actualDocument.OperationType;
+            operationType.ToString().ToLowerInvariant().Should().Be(backingDocument["operationType"].AsString);
+
+            if (operationType == ChangeStreamOperationType.Invalidate)
```

Comment:
`Invalidate` contains only `operationType` and `clusterTime`

Reply:
Ack